tuning/utils/config_utils.py

```diff
@@ -21,20 +21,40 @@ def update_config(config, **kwargs):
                         # In case of specialized config we can warm user
                         print(f"Warning: {config_name} does not accept parameter: {k}")
 
+def create_tuning_config(peft_method, **kwargs):
+    """Create peft_config Tuning config
+       Args:
+           peft_method: str 
+              lora, pt or None
+           kawrgs: parameters to initialize library configs with
+        Return:
+           peft_config.LoraConfig | peft_config.PromptTuningConfig | None
+    """
+    assert peft_method in [None, "lora", "pt", "None"], \
+        f"peft config {peft_method} not defined in peft.py"
+    if peft_method == "lora":
+        tune_config = peft_config.LoraConfig()
+        update_config(tune_config, **kwargs)
+    elif peft_method == "pt":
+        tune_config = peft_config.PromptTuningConfig()
+        update_config(tune_config, **kwargs)
+    else:
+        tune_config = None # full parameter tuning
+    return tune_config
 
-def get_peft_config(train_config, kwargs):
-    assert train_config.peft_method in [None, "lora", "pt"], \
-        f"peft config {train_config.peft_method} not defined in peft.py"
 
-    if train_config.peft_method == "lora":
-        config = lora_config()
-        update_config(config, **kwargs)
-        peft_config = LoraConfig(**asdict(config))
-    elif train_config.peft_method == "pt":
-        config = prompt_tuning_config()
-        update_config(config, **kwargs)
-        peft_config = PromptTuningConfig(**asdict(config))
+def get_hf_peft_config(task_type, tuning_config):
```

Comment:
since train() now accepts the peft_config objects, it makes sense to use those to get the corresponding HF peft config. 
Earlier functionality to pass all kwargs has been moved to `create_tuning_config` utility which can be combined with `get_hf_peft_config` if needed.